shared/team-building/service-tab-bar.desktop.tsx

```diff
@@ -128,23 +80,16 @@ const styles = Styles.styleSheetCreate({
     backgroundColor: Styles.globalColors.blue,
     height: 2,
   },
-  clickableServiceIcon: {
-    flex: 1,
-  },
   inactiveTabBar: {
-    backgroundColor: Styles.globalColors.black_10,
-    height: 1,
+    borderBottomWidth: 1,
+    borderColor: Styles.globalColors.black_10,
+    height: 2,
   },
-  pendingIcon: Styles.platformStyles({
-    isElectron: {height: 10, width: 10},
-    isMobile: {height: 17, width: 17},
-  }),
+  pendingIcon: {height: 10, width: 10},
   resultCount: {},
```

Comment:
do we need this empty object for anything?

Reply:
We don't. Will kill it in the next PR